apps/frontend/src/app/Components/LevelSelect.tsx

```diff
@@ -51,8 +51,9 @@ export default function LevelSelect({
       if (talent) {
         const clampedTalent = { ...talent }
```

Comment:
Looks like we can reuse the validateTalent function here?

Reply:
Yep , added in https://github.com/frzyc/genshin-optimizer/pull/1454/commits/64c4f410f4d72a9b642f9e177aeb3b44a749901c . Skipping ternary operator to favour readability